events.lisp

```diff
@@ -243,7 +245,9 @@ The Caller is responsible for setting up the input focus."
                ((eq cmd t))
                (cmd
                 (unmap-message-window (current-screen))
-                (eval-command cmd t) t)
+                (let ((*current-key-seq* key-seq))
+                  (eval-command cmd t))
```

Comment:
Can we get away with a simple setq instead?

Reply:
Possibly - dynamic binding seemed safer given my limited understanding of reentrancy in the event handling code in StumpWM.

For instance, I see `stumpwm:push-event` which seems to serialize event handlers in the `main-thread` (if I understand this correctly), so the `setq` would probably work fine - but if there's ever a future change to run the events in different threads the dynamic binding would be safer. HTH.

Reply:
Also, there shouldn't be a huge (performance or memory) cost to this specific use of dynamic binding.